packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -3761,6 +3761,11 @@ export const audiusBackend = ({
     return audiusLibs
   }
 
+  async function getAudiusLibsTyped() {
+    await waitForLibsInit()
+    return audiusLibs as AudiusLibsType
+  }
+
```

Comment:
do we need a separate function for this?

Reply:
yeah sadly when i type the real one there are like 100+ type errors :/ should resolve this at some point though